docs/src/main/sphinx/installation/rpm.rst

```diff
@@ -0,0 +1,91 @@
+================
+RPM package
+================
+
+Users can install Trino using the RPM Package Manager (RPM) on Linux
+distributions that support RPM.
+
+The RPM archive contains the application, all plugins, the necessary default
+configuration files, default setups, and integration with the operating system
+to start as a service.
+
+Installing Trino
+----------------
+
+Download the Trino server RPM package :maven_download:`server-rpm`. Use the
+``rpm`` command to install the package:
+
+.. code-block:: text
+
+    rpm -i trino-server-rpm-*.rpm --nodeps
+
+Installing the :ref:`required Java and Python setup <requirements>` is typically
+managed separately.
+
+Service script
+--------------
+
+The RPM installation deploys a service script configured with ``chkconfig`` so
+that the service can be started automatically on operating system boot. After
+installation, you can manage the Trino server with the ``service`` command:
+
+.. code-block:: text
+
+    service trino [start|stop|restart|status]
+
+* ``start`` Starts the server as a daemon and returns the process ID.
+* ``stop`` Shuts down the server started with either ``start`` or ``run``. Sends
+  the SIGTERM signal.
+* ``restart`` Stops and then starts a running server, or starts a stopped
+  server, assigning a new process ID.
+* ``status`` Prints a status line, either *Stopped pid* or *Running as pid*.
+
+Installation directory structure
+--------------------------------
+
+The RPM package places the various files used by Trino in accordance with the
+Linux Filesystem Hierarchy Standard. This differs from the default tarball
+installation of Trino, where all folders are in the installation directory. For
+example, with a ``tar.gz`` file, configuration files are located by default in the
```

Comment:
I'm not sure it's relevant to compare to the tgz. It would be simpler to just point to the configuration directory.

Reply:
well.. maybe.. the thing is that this relies on the launcher script setup as done in chkconfig to set these paths..

Reply:
so maybe we can leave the mention of the tarball out but clarify that the launcher script parameters are set for these paths to work

Reply:
Isn't the launcher an implementation detail in the RPM? Users don't need to interact with it at all.

Reply:
pretty much .. so maybe we can hide that fact and just talk about the folders..